src/enums/lease.ts

```diff
@@ -18,7 +18,8 @@ export enum LeaseAgreementStatus {
 }
 
 export enum AddedLeaseAgreementDocument {
-  ADDENDUM = 'addendum',
+  ADDENDUM = 'ADDENDUM',
+  REQUESTED_ADDENDUM = 'REQUESTED_ADDENDUM',
```

Comment:
Remove this. Add a property to `ILeaseAgreementDocument` instead to keep track of the owner of the document - perhaps `ownerId`